package.json

```diff
@@ -11,7 +11,26 @@
     "build": "strapi build",
     "develop": "strapi develop",
     "start": "strapi start",
-    "strapi": "strapi"
+    "strapi": "strapi",
+    "test": "jest --forceExit --detectOpenHandles",
```

Comment:
**issue:** running `yarn test` in my dev environment spits out an error at the end:

```
Jest has detected the following 1 open handle potentially keeping Jest from exiting:

  ●  TTYWRAP
```

All the tests pass but this may be confusing to less experienced devs. Is it easy to fix this?

Reply:
Huh, I'm not seeing this, is it a gitpod thing maybe?